mqtt-iwatch-ssl/README.md

```diff
@@ -0,0 +1,345 @@
+# Eclipse-Ditto-MQTT-iWatch-SSL
+This example presents how to configure Ditto to be able update things via MQTT with SSL. We will create an iWatch from a WoT TM (Web of Things Thing Model) in this example. Our Digital Twin will be updated via MQTT, using synthetic data.
+In this tutorial, we will ensure data security in traffic with two-factor authentication (`Username and Password` + `Certificates`). 
```

Comment:
Are you sure that really both authentication mechanisms are used at the same time?
I would expect that only the client certificate is used.
I would not expect that the mqtt broker supports 2 factor auth in that way..

Reply:
I changed the wording. I think that dual authentication is more accurate.